src/components/PostInfo/components/Save.tsx

```diff
@@ -80,8 +81,8 @@ export const Save = ({
             }}
             data-qa={qa}
             aria-pressed={hasUserLike}
-            aria-label={i18n(Keyset.Save)}
-            aria-describedby={titleElementId}
+            aria-label={isNumericTitle ? i18n(Keyset.Save) : undefined}
+            aria-describedby={isNumericTitle ? titleElementId : undefined}
```

Comment:
Explanation: there is no need to add a label if `title` is text. At the same time when `title` is a number it is not very clear what the button does, so we add `aria-label`